ntbs-service/wwwroot/source/Components/ValidateDate.ts

```diff
@@ -31,15 +30,15 @@ const ValidateDate = Vue.extend({
                 "RequestVerificationToken": (<HTMLInputElement>document.querySelector('[name="__RequestVerificationToken"]')).value
             }
 
-            axios.post(`/${this.$props.model}s/Edit/Validate${this.$props.model}Date?key=${this.$props.property}&day=${date.day}&month=${date.month}&year=${date.year}`, 
+            axios.post(`${getValidationBaseBath(this.$props.model)}Date?key=${this.$props.property}&day=${date.day}&month=${date.month}&year=${date.year}`, 

```

Comment:
Obviously this is related to moving everything to `Validate`, but if we don't, one way to simplify this would be to keep these links relative (to the current page)?